src/markup/codemirror/files-upload-plugin/plugin.ts

```diff
@@ -148,19 +148,25 @@ class FileUploadPresenter {
     private async formatFileMarkup(res: FileUploadResult) {
         const fileName = res.name ?? this.file.name ?? '';
 
-        let markup: string;
+        let markup = `![${fileName}](${res.url}`;
+
         if (isImageFile(this.file)) {
             if (this.needDimmensionsForImages) {
                 try {
-                    let {height} = await getImageDimensions(this.file);
-                    height = Math.min(height, IMG_MAX_HEIGHT);
-                    markup = `![${fileName}](${res.url} =x${height})`;
+                    const fileSize = await getImageDimensions(this.file);
+                    const {width, height} = getProportionalSize({
```

Comment:
Let's add an option in the [experiment config](https://github.com/gravity-ui/markdown-editor/blob/main/src/bundle/types.ts#L51) and use it both in WYSIWYG and in markup mode